ledger/participant-integration-api/src/main/resources/db/migration/oracle-appendonly/V1__Append_only_schema.sql

```diff
@@ -128,13 +128,13 @@ CREATE TABLE participant_command_completions
     deduplication_duration_nanos    NUMBER,
     deduplication_start             NUMBER,
 
-    submitters                  CLOB NOT NULL CONSTRAINT ensure_json_submitters CHECK (submitters IS JSON),
-    command_id                  NVARCHAR2(1000) NOT NULL,
+    submitters                      CLOB NOT NULL CONSTRAINT ensure_json_submitters CHECK (submitters IS JSON),
+    command_id                      NVARCHAR2(1000) NOT NULL,
 
-    transaction_id              NVARCHAR2(1000), -- null for rejected transactions and checkpoints
-    rejection_status_code       INTEGER,         -- null for accepted transactions and checkpoints
-    rejection_status_message    CLOB,            -- null for accepted transactions and checkpoints
-    rejection_status_details    BLOB             -- null for accepted transactions and checkpoints
+    transaction_id                  NVARCHAR2(1000), -- null for rejected transactions and checkpoints
+    rejection_status_code           INTEGER,         -- null for accepted transactions and checkpoints
+    rejection_status_message        CLOB,            -- null for accepted transactions and checkpoints
+    rejection_status_details        BLOB             -- null for accepted transactions and checkpoints
```

Comment:
Preserving the existing format but I think it's more practical not to column-align (maybe a separate change).

Reply:
Did you actually change anything in this file except whitespace?

Reply:
No, just whitespace. I'll revert so we don't have to create a new migration.

Reply:
22cd962